sympy/matrices/eigen.py

```diff
@@ -146,8 +146,8 @@ def _eigenvals(
     Notes
     =====
 
-    Eigenvalues of a matrix `A` can be computed by solving a matrix
-    equation `\det(A - \lambda I) = 0`
+    Eigenvalues of a matrix ``A`` can be computed by solving a matrix
+    equation ``\det(A - \lambda I) = 0``
```

Comment:
These looks like LaTeX math macros than python code. So they should be changed to dollars instead.